src/components/CasesCard/index.js

```diff
@@ -27,19 +27,19 @@ const Severe = styled(Tile)`
   border-bottom: ${(props) => (props.selected ? '0.625rem' : '0.3125rem')} solid;
   border-color: ${(props) =>
     props.riskType === 'high'
-      ? '#eb2f2f'
+      ? '#FF3636'
```

Comment:
better to store these values in constants. In the future if we decide to change the color, then we would only have to change it one place

Reply:
resolved